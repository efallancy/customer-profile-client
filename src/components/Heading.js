import styled from 'styled-components';

export const PageHeading = styled.h1`
  font-size: 48px;
  color: #121958;
  letter-spacing: 0.16px;
`;

export const TitleHeading = PageHeading.withComponent('h3').extend`
  font-size: 28px;
  margin: 0 0 1rem;
`;

export const ErrorHeading = styled.p`
  color: #FF4040;
  font-size: 15px;
`;

export default TitleHeading;