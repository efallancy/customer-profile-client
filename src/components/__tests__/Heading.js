import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { PageHeading, TitleHeading, ErrorHeading } from '../Heading';

describe('Heading', () => {
  it('Page heading renders correctly', () => {
    const pageHeading = renderer.create(<PageHeading />).toJSON();
    expect(pageHeading).toMatchSnapshot();
  });

  it('Title heading renders correctly', () => {
    const titleHeading = renderer.create(<TitleHeading />).toJSON();
    expect(titleHeading).toMatchSnapshot();
  });

  it('Error heading renders correctly', () => {
    const errorHeading = renderer.create(<ErrorHeading />).toJSON();
    expect(errorHeading).toMatchSnapshot();
  });
});
