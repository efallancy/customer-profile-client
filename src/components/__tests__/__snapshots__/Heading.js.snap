// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading Error heading renders correctly 1`] = `
.c0 {
  color: #FF4040;
  font-size: 15px;
}

<p
  className="c0"
/>
`;

exports[`Heading Page heading renders correctly 1`] = `
.c0 {
  font-size: 48px;
  color: #121958;
  -webkit-letter-spacing: 0.16px;
  -moz-letter-spacing: 0.16px;
  -ms-letter-spacing: 0.16px;
  letter-spacing: 0.16px;
}

<h1
  className="c0"
/>
`;

exports[`Heading Title heading renders correctly 1`] = `
.c0 {
  font-size: 48px;
  color: #121958;
  -webkit-letter-spacing: 0.16px;
  -moz-letter-spacing: 0.16px;
  -ms-letter-spacing: 0.16px;
  letter-spacing: 0.16px;
  font-size: 28px;
  margin: 0 0 1rem;
}

<h3
  className="c0"
/>
`;
